chainerx_cc/chainerx/dtype.h

```diff
@@ -207,6 +207,26 @@ auto VisitIntegralDtype(Dtype dtype, F&& f, Args&&... args) {
     }
 }
 
+// Invokes a function by passing PrimitiveType<T> corresponding to given Integral dtype value.
+// See VisitDtype for more detail.
+template <typename F, typename... Args>
+auto VisitShiftDtype(Dtype dtype, F&& f, Args&&... args) {
```

Comment:
It seems we should give better name.

Reply:
Perhaps we should name this as `VisitIntegralDtype` and rename what is currently called `VisitIntegralDtype` to something else.
So, in this PR, could you tentatively write a TODO about that?
Something like "Rename this to VisitIntegralDtype and current VisitIntegralDtype to something else."